### PR TITLE
fixed bug of not scrolling to top on change of page react router

### LINKED ISSUE #1164: React Router dom problem of not scrolling to top on route changes
**Describe the bug**
currently when user change from one page to another scrollbar remain at same position. it doesn't going at top.

**To Reproduce**

1. Go to essential page(https://www.covid19india.org/essentials)
2. from dropdown select maharastra and click search.
3. Scroll down to bottom
4. then go to deep dive / Home(https://www.covid19india.org/deepdive / https://www.covid19india.org)
5. See error

**Expected behavior**
it should open page with scroll at top.

**Screenshots**
scroll bug

**Desktop (please complete the following information):**

- OS: [all os]
- Browser [sll browser]
- Version [all version]

### PR DESCRIPTION
**Pull request by Binu42**
**Description of PR**
* fixed, on change of page scroll does not going to top.

**Type of PR**

- [x] Bugfix

**Relevant Issues**  
Fixes #1164 

**Checklist**

- [x] Compiles and passes lint tests
- [x] Properly formatted
- [x] Tested on desktop

**Screenshots**
#### demo after fixed.
fixed scroll

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -2,8 +2,17 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import {BrowserRouter as Router} from 'react-router-dom';
+import ScrollToTop from './utils/ScrollToTop';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render(
+  <Router>
+    <ScrollToTop>
```
Comment (karangarg45) [inline src/index.js:10]: you don't need to wrap your App in ScrollToTop component. There is nothing wrong with it, just a suggestion..

Comment (karangarg45) [reply src/index.js:10]: You can just return null if you decide not to wrap it

Comment (Binu42) [reply src/index.js:10]: yeah, thanks for suggestion

**Review by jeremyphilemon — APPROVED** [review]
Looks good! Thank you :)

**Pull request merged**